src/applications/vaos/tests/containers/DateTimeRequestPage.unit.spec.jsx

```diff
@@ -47,13 +47,14 @@ describe('VAOS <DateTimeRequestPage>', () => {
   it('should not submit form with validation error', () => {
     const onCalendarChange = sinon.spy();
     const routeToNextAppointmentPage = sinon.spy();
+    const data = {
+      calendarData: { currentlySelectedDate: '2020-12-20', error: 'test' },
```

Comment:
Could we set this date (and the ones below) to today's date, plus a year (or some other interval)? That seems less likely to maybe cause a time-based unit test failure down the road.